### PR TITLE
feat: Add TESWorldspace.h, decoding of TESObjectCell, GetParentWorldSpace for TESObjectRef*

### PR DESCRIPTION
**Pull request by TommInfinite**

### CODE REVIEW

File: CommonLibSF/include/RE/T/TESObjectCELL.h [hunk 1]
```diff
@@ -12,7 +13,7 @@ namespace RE
 	public:
 		SF_RTTI_VTABLE(TESObjectCELL);
 		SF_FORMTYPE(CELL);
-
-		//
+		bool           pad[0xD8];
+		TESWorldSpace* CellWorldspace;
```
Comment (powerof3) [inline CommonLibSF/include/RE/T/TESObjectCELL.h:17]: please use camelCase for member variables (ie. `cellWorldspace`) and add a comment with its offset.

Comment (TommInfinite) [reply CommonLibSF/include/RE/T/TESObjectCELL.h]: done, thanks for the comments.

**Comment by ThirdEyeSqueegee** [comment]
Just fyi, you don't need to manually update Starfield.h--that happens automatically during build configuration

File: CommonLibSF/include/RE/T/TESObjectCELL.h [hunk 2]
```diff
@@ -12,7 +13,7 @@ namespace RE
 	public:
 		SF_RTTI_VTABLE(TESObjectCELL);
 		SF_FORMTYPE(CELL);
-
-		//
+		uint8_t        pad[0xD8];
```
Comment (ThirdEyeSqueegee) [inline CommonLibSF/include/RE/T/TESObjectCELL.h:16]: `std::uint8_t` for consistency, all `std` namespace members are written out fully qualified in CommonLib

File: CommonLibSF/include/RE/W/TESWorldSpace.h [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#pragma once
```
Comment (ThirdEyeSqueegee) [inline CommonLibSF/include/RE/W/TESWorldSpace.h:1]: Add an empty line after the pragma

**Review by ThirdEyeSqueegee — CHANGES_REQUESTED** [review]

File: CommonLibSF/include/RE/T/TESObjectCELL.h [hunk 4]
```diff
@@ -12,7 +13,7 @@ namespace RE
 	public:
 		SF_RTTI_VTABLE(TESObjectCELL);
 		SF_FORMTYPE(CELL);
```
Comment (ThirdEyeSqueegee) [inline CommonLibSF/include/RE/T/TESObjectCELL.h:15]: Add an empty line after the `SF_FORMTYPE` macro

**Pull request closed without merging**